python/cac_tripplanner/cac_tripplanner/settings.py

```diff
@@ -0,0 +1,96 @@
+"""
+Django settings for cac_tripplanner project.
```

Comment:
Typically we will template this file in ansible, so we can control dev/staging/prod variables at the group_vars level. Should we do that here as well?